scripts/delete-unused-users.py

```diff
@@ -10,60 +10,138 @@
 make sure they aren't active right now. This will require users to
 log in again the next time they use the hub, but that's probably
 ok.
+
+Core functionality from @minrk:
+https://discourse.jupyter.org/t/is-there-a-way-to-bulk-delete-old-users/20866/3
 """
 import argparse
-from jhub_client.api import JupyterHubAPI
-from dateutil.parser import parse
 import asyncio
 from datetime import timedelta, datetime
+from dateutil.parser import parse
+from jhub_client.api import JupyterHubAPI
```

Comment:
Not critical: per [PEP8](https://peps.python.org/pep-0008/#imports), third party imports should follow those from standard library.